packages/graphql-api/package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@graphql-tools/merge": "6.2.5",
     "@graphql-tools/utils": "7.0.2",
+    "graphql": "14.6.0",
```

Comment:
Multiple graphql versions usually creates problem, I think it would be better to have it as a "peerDependencies".

Reply:
@pradel Done!